maraboupy/MarabouNetworkNNet.py

```diff
@@ -141,18 +281,108 @@ def read_nnet(self, file_name):
             self.weights = weights
             self.biases = biases
 
-            # Convert input bounds from their original values to normalized values if normalization is not being incorporated
             if not self.normalize:
                 for i in range(self.inputSize):
                     self.inputMinimums[i] = (self.inputMinimums[i] - self.inputMeans[i]) / self.inputRanges[i]
                     self.inputMaximums[i] = (self.inputMaximums[i] - self.inputMeans[i]) / self.inputRanges[i]
 
+
+
+    def writeNNet(self, file_name: str):
+        """Write network data into the .nnet file format
+
+        Args:
+            file_name (str): File where the network will be written
+
+        :meta private:
```

Comment:
The ":meta private:" line makes the method private so that the method won't appear in documentation, which is for methods that ideally a general user wouldn't use. However, it seems like this might be a useful for a user to have available so they can write their own NNet file? If so, I'd recommend removing the ":meta private: line.